stimela/singularity.py

```diff
@@ -31,7 +31,7 @@ def pull(image, name, docker=True, directory=".", force=False):
         pull an image
     """
     if docker:
-        fp = "docker://{0:s}".format(image)
+        fp = "docker://quay.io/{0:s}".format(image)
```

Comment:
Running with singularity without the image on disk gives the following when trying to download:
`# FATAL:   While making image from oci registry: error fetching image to cache: failed to get checksum for docker://stimela/msutils:1.6.9: Error reading manifest 1.6.9 in docker.io/stimela/msutils: manifest unknown: manifest unknown`

Reply:
Let's not hardcode the repositroy. See fix at e699752a..53ab7141